_pages/incident-reports.md

```diff
@@ -35,7 +35,7 @@ certain events caused the incident, those events should also be added
 
 Analyze the **factors** that contributed to the incident. Here it's important
 to emphasize the [Retrospective Prime
-Directive](http://www.retrospectives.com/pages/retroPrimeDirective.html);
+Directive](https://web.archive.org/web/20171017112557/http://www.retrospectives.com/pages/retroPrimeDirective.html);
```

Comment:
@alexsoble @LindsayYoung is there a more up-to-date link we could use rather than using the web archive?